### PR TITLE
Adding ParallelMaximumCliques.jl dependency LibPMC

### PR DESCRIPTION
**Pull request by dev10110**

### CODE REVIEW

File: P/PMC/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using BinaryBuilder
+
+name = "LibPMC"
```
Comment (giordano) [inline P/PMC/build_tarballs.jl:3]: This name must match exactly the name of the diretory

Comment (dev10110) [reply P/PMC/build_tarballs.jl:3]: of the directory in `Yggdrasil`?

Comment (giordano) [reply P/PMC/build_tarballs.jl:3]: Yes, the directory is called `PMC, the package is called `LibPMC`, they should match up to the case.  Also, please read the documentation: https://docs.binarybuilder.org/stable/building/#Name

File: L/LibPMC/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using BinaryBuilder
+
+name = "LibPMC"
```
Comment (giordano) [inline L/LibPMC/build_tarballs.jl:3]: Quoting from the [documentation section](https://github.com/JuliaPackaging/Yggdrasil/pull/6026#discussion_r1056998837) I [suggested reading](https://github.com/JuliaPackaging/Yggdrasil/pull/6026#discussion_r1056998837):

>  It should be the name of the upstream package, not for example that of a specific library or executable provided by it

Looks like a more appropriate name would be PMC?

Comment (dev10110) [reply L/LibPMC/build_tarballs.jl]: ok sure, I've changed it.

**Comment by dev10110** [comment]
@giordano looks like all the checks have passed - do you know what happens next to integrate this package into Julia's package registry?

File: P/PMC/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+rm -rf build
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
+# platforms = [Platform("x86_64", "macOS")]
```
Comment (giordano) [inline P/PMC/build_tarballs.jl:21]:
```suggestion
```

File: P/PMC/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+platforms = expand_cxxstring_abis(platforms)
+
+products = [
+    LibraryProduct("libpmc", :libpmc),
+]
+
+dependencies = [
```
Comment (giordano) [inline P/PMC/build_tarballs.jl:28]:
```suggestion
dependencies = Dependency[
```

File: P/PMC/build_tarballs.jl (hunk 3 above)
Comment (dev10110) [reply P/PMC/build_tarballs.jl]: updated.

File: P/PMC/build_tarballs.jl (hunk 4 above)
Comment (dev10110) [reply P/PMC/build_tarballs.jl]: updated.

**Comment by dev10110** [comment]
Thanks @giordano!

**Pull request merged**